django/thunderstore/community/templates/community/packagelisting_list.html

```diff
@@ -3,11 +3,12 @@
 {% load arrow %}
 {% load cache_until %}
 {% load qurl %}
+{% load get_urls %}
 
 {% block title %}{{ page_title }}{% endblock %}
 
 {% block content %}
-{% cache_until "any_package_updated" "mod-list" 300 page_obj.number cache_vary %}
+{% cache_until "any_package_updated" "mod-list" 300 page_obj.number cache_vary community_identifier use_old_urls %}
```

Comment:
I'm fairly sure the `community_identifier` is already included in the `cache_vary` context variable and could be omitted.

Reply:
Yep it is, just didn't connect the dots, that both the views cache vary and these args are used in the same cache generation, I'll remove it since it's obsolete 👌